document/CoreFormat.mdk

```diff
@@ -2425,10 +2425,11 @@ This allows tools to include information about the encountered condition that is
 ### General { #exception-General }
 
 An `exception` object describes a runtime exception encountered in the course of executing an analysis tool.
+This includes signals in operating systems such as POSIX.
 
 ### `type` property { #exception-type }
 
-An `exception` object shall contain a property named `type` whose value is a string containing the fully qualified
+An `exception` object should contain a property named `type` whose value is a string containing the fully qualified
```

Comment:
I guess it was wrong of me to overload "exception type" with "signal number". What if we had a dedicated "signalNumber" property for POSIX? The thing is that "SIGFPE" is easier to read than... whatever its number happens to be in signal.h. I think having both an integer signalNumber property and a string signalName property is overdoing it... I guess I could live with

string type: dedicated to exception type
integer signalNumber: dedicated to POSIX signals.

Thoughts, either of you?